lemur/tests/test_integration_db_and_models.py

```diff
@@ -0,0 +1,239 @@
+# This file consists of testing for the creation of objects and their
+# relationship in db
+import sys
+sys.path.append('../..')
```

Comment:
Why is this needed? This is going to change your path each time this gets run, won't it?